app/code/Magento/InventoryShipping/view/adminhtml/ui_component/inventory_shipping_source_selection_form.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<form xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" component="Magento_InventoryShipping/js/form/form" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
+    <argument name="data" xsi:type="array">
+        <item name="config" xsi:type="array">
+            <item name="sourceSelectionUrl" xsi:type="url" path="*/*/ProcessAlgorithm"/>
+        </item>
+        <item name="js_config" xsi:type="array">
+            <item name="provider" xsi:type="string">inventory_shipping_source_selection_form.inventory_shipping_source_selection_form_data_source</item>
+        </item>
+        <item name="label" xsi:type="string" translate="true">Source Selection</item>
+        <item name="template" xsi:type="string">templates/form/collapsible</item>
+    </argument>
+    <settings>
+        <dataScope>data</dataScope>
+        <namespace>inventory_shipping_source_selection_form</namespace>
+        <deps>
+            <dep>inventory_shipping_source_selection_form.inventory_shipping_source_selection_form_data_source</dep>
+        </deps>
+        <buttons>
+            <button name="back">
+                <url path="sales/order/index"/>
+                <class>back</class>
+                <label translate="true">Back</label>
+            </button>
+            <button name="reset">
+                <class>reset</class>
+                <label translate="true">Reset</label>
+            </button>
+            <button name="save" >
+                <class>save primary</class>
+                <label translate="true">Create Shipment</label>
+            </button>
+            <button name="source_selection_algorithms" class="Magento\InventoryShipping\Ui\Component\Control\SourceSelection\AlgorithmSelectionButton"/>
+        </buttons>
+    </settings>
+    <dataSource name="inventory_shipping_source_selection_form_data_source" component="Magento_Ui/js/form/provider">
+        <settings>
+            <validateUrl path="*/*/validate"/>
+            <submitUrl path="admin/order_shipment/new"/>
+        </settings>
+        <dataProvider class="Magento\InventoryShipping\Ui\DataProvider\SourceSelectionDataProvider" name="inventory_shipping_source_selection_form_data_source">
+            <settings>
+                <requestFieldName>order_id</requestFieldName>
+                <primaryFieldName>order_id</primaryFieldName>
+            </settings>
+        </dataProvider>
+    </dataSource>
+    <fieldset name="general" sortOrder="10">
+        <settings>
+            <dataScope />
+            <label />
+            <collapsible>false</collapsible>
+            <opened>true</opened>
+        </settings>
+        <field name="order_id" formElement="hidden">
```

Comment:
I think this hidden field can be removed. order_id will always present inside provider, no reason to declare it.